presentation/common/src/presentation-common/content/Descriptor.ts

```diff
@@ -130,6 +160,29 @@ export interface DescriptorJSON {
   filterExpression?: string;
 }
 
+/**
+ * Serialized [[Descriptor]] JSON representation.
+ * @beta
+ */
+export interface CompressedDescriptorJSON {
```

Comment:
IMO it would be better to omit / add attributes from / to `DescriptorJSON` instead of re-defining everything.

Reply:
Updated it